gulpfile.js

```diff
@@ -121,7 +121,6 @@ function writeServiceWorkerFile() {
       `${rootDir}/js/*.js`,
       `${rootDir}/css/*.css`,
       `${rootDir}/bower_components/**/webcomponents-lite.min.js`,
-      `${rootDir}/bower_components/highlight/highlight.js`,
```

Comment:
Going forward, I suggest doing all highlighting on the server side (since I see little benefit of making the client doing the work).